package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "start": "webpack serve --open",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack --mode developmen"
```

Comment:
[OPTIONAL] Good job with setting up the webpack. But let's fix the type here. "dev": "webpack --mode developmen**t**"